pfcpiface/node.go

```diff
@@ -55,6 +55,23 @@ func (node *PFCPNode) handleNewPeers() {
 	lAddrStr := node.LocalAddr().String()
 	log.Infoln("listening for new PFCP connections on", lAddrStr)
 
+	for _, peer := range node.upf.peers {
+		n4DstIP, err := getRemoteIP(peer)
+		if err != nil {
+			log.Infoln("Failed to establish PFCP connection to peer ", peer)
```

Comment:
If the operation is failed but we decide to continue, we should use `log.Warn`.

Reply:
Okay, done